### PR TITLE
Virt Best Practices: add scsi persistent reservation on multipathed

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Description

Describe the overall goals of this pull request.
multipath iSCSI update to virt best practices

### PR creator: Are there any relevant issues/feature requests?

* bsc#1210642

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.5
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [x] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Review by morecache — APPROVED** [review]
looks good to me

**Comment by tbazant** [comment]
@morecache i've restructured the doc a bit, can you have a look once more pls?

File: xml/art_virtualization-best-practices.xml [hunk 1]
```diff
@@ -3241,6 +3241,265 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
...
+          scenario, because it makes sure that the storage paths are identical
+          between the source and destination hosts.
+        </para>
+        <para>
+          Find more information about multipath in
+          <xref linkend="sec-multipath-alias"/>.
+        </para>
+      </note>
+      <procedure>
+        <title>Adding an SCSI persistent reservation in a &vmguest; against the related multipathed device in the &vmhost;:</title>
```
Comment (dariavladykina) [inline xml/art_virtualization-best-practices.xml:3281]:
```suggestion
        <title>Adding a SCSI persistent reservation in a &vmguest; against the related multipathed device in the &vmhost;:</title>
```

File: xml/art_virtualization-best-practices.xml [hunk 2]
```diff
@@ -3241,6 +3241,265 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
...
+        <step>
+          <para>
+            In the &vmhost;, create multipath environment. For more
```
Comment (dariavladykina) [inline xml/art_virtualization-best-practices.xml:3284]:
```suggestion
            In the &vmhost;, create a multipath environment. For more
```

File: xml/art_virtualization-best-practices.xml [hunk 3]
```diff
@@ -3241,6 +3241,265 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
...
+        </step>
+        <step>
+          <para>
+            In the &vmhost;, configure the
+            <literal>&lt;reservations/></literal> sub-element of the
+            <literal>&lt;source/></literal> element of the
+            <literal>&lt;disk/></literal> element for the passed-through lun in
+            your &libvirt; domain configuration. Refer to
+            <link xlink:href="https://libvirt.org/formatdomain.html">&libvirt;
+            Domain XML format</link>
```
Comment (dariavladykina) [inline xml/art_virtualization-best-practices.xml:3298]:
```suggestion
            Domain XML format</link>.
```

File: xml/art_virtualization-best-practices.xml [hunk 4]
```diff
@@ -3241,6 +3241,265 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
...
+          </para>
+        </step>
+        <step>
+          <para>
+            In the &vmguest;, install the <package>sg3_utils</package> package
+            and reserve the SCSI disks on-demand by the
```
Comment (dariavladykina) [inline xml/art_virtualization-best-practices.xml:3304]:
```suggestion
            and reserve the SCSI disks on demand by the
```

File: xml/art_virtualization-best-practices.xml [hunk 5]
```diff
@@ -3241,6 +3241,265 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
...
+    &lt;backingStore/&gt;
+    &lt;target dev='sda' bus='scsi'/&gt;
+    &lt;alias name='scsi0-0-0-0'/&gt;
+    &lt;address type='drive' controller='0' bus='0' target='0' unit='0'/&gt;
+  &lt;/disk&gt;</screen>
+          </step>
+          <step>
+            <para>
+              In the &vmguest;, reserve the scsi disk, for example,
+              <literal>sda</literal>, with the key <literal>123abc</literal>
```
Comment (dariavladykina) [inline xml/art_virtualization-best-practices.xml:3458]:
```suggestion
              <literal>sda</literal>, with the key <literal>123abc</literal>.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some small suggestions. Thanks!

**Pull request merged**